### PR TITLE
AllNft calls & technical debt

### LINKED ISSUE #719: All Nfts call error
image

All Nft call is erroring saying some value is null, will look into this. The endpoint was deprecated afaik and we need to put the new one in.

### PR DESCRIPTION
**Pull request by sneakzttv**
All NFT calls are now deserializing into a nice new model from the new API and displayed in the console when called. For #719
(These were working fine until this morning when the api team closed the endpoint, I guess this comes with 2.5.2 now.
I'm removing the issue from the PR, it won't be resolved for at least a few weeks as the API team is transitioning to turbosquid.

Webgl templates now only sync check if webgl is active as the editor platform when opening server settings as per #680

Added a wallet exception and made it fire when a wallet has issues connecting from the login scene as per #680

### CODE REVIEW

File: Packages/io.chainsafe.web3-unity/Runtime/Scripts/Model/AllNftModel.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace ChainSafe.Gaming.UnityPackage.Model
+{
+    public class AllNftModel
```
Comment (kantagara) [inline Packages/io.chainsafe.web3-unity/Runtime/Scripts/Model/AllNftModel.cs:6]: Was this necessary to be a class?

Comment (sneakzttv) [reply Packages/io.chainsafe.web3-unity/Runtime/Scripts/Model/AllNftModel.cs:6]: yes, allowing it do be deserialized into a class gives the users heaps of freedom when navigating the json responses. It's how we've always done it with the API calls, if you check the models in the runtime folder you'll see. If you'd like it done another way just let me know <3

Comment (sneakzttv) [reply Packages/io.chainsafe.web3-unity/Runtime/Scripts/Model/AllNftModel.cs:6]: Doesn't really matter anyway as it now needs to be redone with the API endpoint being decommissioned this morning :|

**Review by juans-chainsafe — APPROVED** [review]
All good!

- AllNft 🕐 (didn't test it)
- WebGL templates when I click Save settings ✅ 
- Login exception ✅  (I couldn't make it appear but login works fine)

**Review by robGG1997 — APPROVED** [review]

File: Packages/io.chainsafe.web3-unity/Editor/ServerSettings.cs [hunk 2]
```diff
@@ -204,4 +208,25 @@ private class ValidateProjectIDResponse
...
+            WebGLTemplateSyncStatus.DoesntExist =>
+                EditorUtility.DisplayDialog("web3.unity", "Do you wish to install the web3.unity WebGL templates into your project?", "Yes", "No"),
+            WebGLTemplateSyncStatus.OutOfDate =>
+                EditorUtility.DisplayDialog("web3.unity", "The web3.unity WebGL templates in your project are out of date, would you like to update now?", "Yes", "No"),
+            _ => false,
+        };
+        
+        if (performSync)
+        {
+            WebGLTemplateSync.Syncronize();
```
Comment (RyRy79261) [inline Packages/io.chainsafe.web3-unity/Editor/ServerSettings.cs:228]: Typo (Though I dont see that function within this PR so might be a separate bug fix

```suggestion
            WebGLTemplateSync.Synchronize();
```

**Review by RyRy79261 — APPROVED** [review]
Only the mention of the Typo but otherwise looks good

**Pull request merged**